src/main/java/org/web3j/spring/autoconfigure/Web3jAutoConfiguration.java

```diff
@@ -93,11 +93,11 @@ private static void configureLogging(OkHttpClient.Builder builder) {
             builder.addInterceptor(logging);
         }
     }
-
-
+    
     @Bean
     @ConditionalOnBean(Web3j.class)
```

Comment:
Please update the Bean creation of Web3jHealthEndpointWebExtension to take precedence over SpringBoot 2.0's HealthEndpointWebExtension class, otherwise when using this project as a dependency, SpringBoot's HealthEndpointWebExtension get's used.

Reply:
Thanks for the feedback. I applied your suggestion and added the missing import statement